lib/main.dart

```diff
@@ -1,23 +1,28 @@
-import 'package:flutter/cupertino.dart';
+import 'package:doclense/About.dart';
+import 'package:doclense/Home.dart';
 import 'package:flutter/material.dart';
-import 'SplashScreen.dart';
+import 'package:animated_splash_screen/animated_splash_screen.dart';
+import 'dart:ui';
 
 void main() {
   runApp(new MaterialApp(
     home: MyApp(),
   ));
 }
 
-class MyApp extends StatefulWidget {
-  @override
-  _MyAppState createState() => _MyAppState();
-}
-
-class _MyAppState extends State<MyApp> {
+class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      home: SplashScreen(),
+      debugShowCheckedModeBanner: false,
```

Comment:
Don't modify the icon I will provide the nav icon soon as it does currently not function.